### DISCUSSION TITLE
Touching joysticks - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#46] [General]
# Touching joysticks - Alan Pope's blog

I’ve just returned from an excellent day visting The Cave and the Arcade Archive with friends.
I’ve visited the cave before, and wrote up my experience in Visiting The Cave last year. I decided since (nearly) a year had passed, it was time to visit again.
When I mentioned online to some friends, that I was planning on going, they all booked tickets for the same day. We arrived just before 11am, met the greeters who checked our tickets, showed us up the stairs, and introduced the venue.

https://popey.com/blog/2023/09/touching-joysticks/

**Comment by rhys-the-davies** [comment]
This is not the expose into your sexuality I expected from the title but I did find that this post holds the number 1 webpage rank in Google, after a few untoward videos, for the search term ‘fondling joysticks’ - congratulations?
